controller_manager:
  ros__parameters:
    update_rate: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    steering_controller:
      type: position_controllers/JointGroupPositionController

    drive_controller:
      type: velocity_controllers/JointGroupVelocityController

# =============================

steering_controller:
  ros__parameters:
    joints:
      - steer_joint

drive_controller:
  ros__parameters:
    joints:
      - rear_left_wheel_joint
      - rear_right_wheel_joint